pkg/ansible/watches/watches.go

```diff
@@ -297,7 +298,9 @@ func getAnsibleVerbosity(gvk schema.GroupVersionKind, defValue int) int {
 	))
 	ansibleVerbosity, err := strconv.Atoi(os.Getenv(envVar))
 	if err != nil {
-		log.Info("Failed to parse %v from environment. Using default %v", envVar, defValue)
+		//log.Info("Failed to parse %v from environment. Using default %v", envVar, defValue)
+		log.Info(fmt.Sprintf("Using default value for ansible verbosity %d", defValue))
```

Comment:
```suggestion
		log.Info(fmt.Sprintf("Unable to find a value for %v". Then, using default value for the ansible verbosity %d", envVar, defValue))
```
I think that is good to keep the env var to show how the user can change it.